cleo_plugins/DebugUtils/DebugUtils.cpp

```diff
@@ -43,6 +49,7 @@ class DebugUtils
         }
 
         auto config = GetConfigFilename();
+        configLimitCommand = GetPrivateProfileInt("Limits", "Command", 100000, config.c_str());
```

Comment:
100,000 or 200,000?

Reply:
Excluded legacy scripts and restored restriction to 100000.